packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1444,6 +1446,19 @@ export default class SchedulerTask {
                 status: SchedulerJobStatus.COMPLETED,
             });
         } catch (e) {
+            if (
+                `${e}`.includes('invalid_grant') ||
+                `${e}`.includes('Requested entity was not found')
```

Comment:
what about `Could not find slack installation` ?

Reply:
Good question, I forgot to push that code... :facepalm: